ddtrace/tracer/tracer_test.go

```diff
@@ -1950,9 +1961,23 @@ loop:
 			time.Sleep(time.Millisecond)
 		}
 	}
+	as := &aggregableSpan{
+		key: aggregation{
+			Name: "http.request",
+		},
+		// Start must be older than latest bucket to get flushed
+		Start:    time.Now().UnixNano() - 3*500000,
+		Duration: 1,
+	}
+	c.add(as)
+
 	assert.Len(t, tw.Flushed(), 0)
+	assert.Equal(t, ts.flushed, 0)
+	assert.Len(t, transport.Stats(), 0)
 	tr.flushSync()
 	assert.Len(t, tw.Flushed(), 1)
+	assert.Equal(t, ts.flushed, 1)
+	assert.NotZero(t, transport.Stats())
```

Comment:
Can we assert something more strong than just `NotZero`? Or is the result of `transport.Stats()` not consistent here?